app/assets/javascripts/discourse/app/templates/modal/move-to-topic.hbs

```diff
@@ -68,6 +68,11 @@
           @value={{this.participants}}
           @onChange={{action (mut this.participants)}}
         />
+
+        <label class="checkbox-label">
```

Comment:
Nitpick, for accessibility, it would be nice if we added a `for="chronological-order"` attribute here that matches an `id="chronological-order"` on the input itself. 

Also, can we only show this checkbox if an existing topic is selected? It looks a little odd in the UI before you've picked a topic: 

<img width="653" alt="image" src="https://github.com/discourse/discourse/assets/368961/242e86da-16d2-4c38-aba1-80f898274917">

We also might have to make the modal a bit taller.